views/services.hbs

```diff
@@ -1,26 +1,50 @@
-<section class="">
-  <div class="">
-    <a href="/" aria-label={{text.back}}><img src="./assets/icons/BACK_btn.svg" ></a>
+<section class="white fixed top-0 mb3">
```

Comment:
can we have the "w-100 tc" classes like we do in the home.hbs to centre everything?

I've done it and it works fine, except it means the arrow is in the centre. This can be fixed by using float left or right or whatever, but then it's the same for arabic or english, so one of them is in the wrong place always 😢 

I think probably its better to have the arrow in the centre and make the site as a whole look better than to have the arrow to the right. we can always fix the position using a helper or something.